laws.md

```diff
@@ -84,5 +82,10 @@ New Judges are not bound by the decisions of old Judges. New Judges may, however
 
 This rule takes precedence over every other rule determining the winner.
 
-**301.** Ammendment of rule **201.**: Each player may submit one proposal every twenty-four (24) hours, in any player order. Each proposal will remain for the 24 hour period to be voted on and discussed. At the end of the 24 hour period, the rule is either adopted, or rejected.
+**301.** Each player may submit one proposal every twenty-four (24) hours, in any player order. Each proposal will remain for the 24 hour period to be voted on and discussed. At the end of the 24 hour period, the rule is either adopted, or rejected.
```

Comment:
Maybe this is nitpicky. But this is modifying to rules now. The rule was put in with that text and shouldn't be changed unless you want to propose a rule change.